tests/index.js

```diff
@@ -1,7 +1,24 @@
+const token = process.env.TOKEN
```

Comment:
will these tests call dotenv? if so, are we using `.env.test` for this or just normal `.env`?

Reply:
Mocha (the test software) has a env setup but I just use a dotenv locally for now